packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -54,25 +53,22 @@ export const createPatient = async (
 
   const requestId = uuidv7();
 
-  analytics({
-    distinctId: patient.cxId,
-    event: EventTypes.patientDiscovery,
-    properties: {
-      requestId,
-      patientId: newPatient.id,
-    },
-    apiType: Product.medical,
+  // TODO: add progress to PD #1689
+  const updatedPatient = await storeQueryInit({
```

Comment:
Without going into details, I was just wondering if there was no easy way to set this at `PatientModel.create(patientCreate);`, so we don't create and instantly update the patient

Reply:
I agree. We should be more careful w/ the DB round trips, especially when it means an DB write, even more so when its a DB write w/ DB tx+lock from the app server.

Not that we need to change the whole structure of how we do stuff, but its not something to disregard easily IMO.